synapse/lib/schemas.py

```diff
@@ -97,6 +97,7 @@
         'name': {'type': 'string'},
         'pool': {'type': 'boolean'},
         'doc': {'type': 'string'},
+        'loglevel': {'type': 'string', 'enum': list(s_const.LOG_LEVEL_CHOICES.keys())},
```

Comment:
This will eventually be a part of the storm runtime options, but plumbing now to allow a knob to control the new warning behavior while not digging further tech debt by adding a bool to control the behavior. A slightly stunted implementation for now, but worth being ready for the future.